packages/web/src/assets/styles/colors.css

```diff
@@ -54,7 +53,7 @@
   --search-bar-border: rgba(143, 149, 152, 0.1);
   --search-bar-shadow: rgba(133, 129, 153, 0.15);
 
-  --scrubber-hover: var(--primary-gradient-alt-1);
+  --scrubber-hover: var(--harmony-gradient);
```

Comment:
also not seeing `--harmony-gradient` anywhere..

Reply:
it's defined in the harmony css :)